spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDefinitionController.java

```diff
@@ -16,10 +16,19 @@
 
 package org.springframework.cloud.dataflow.server.controller;
 
+import java.util.ArrayList;
```

Comment:
Not really: https://github.com/spring-projects/spring-security/blob/master/config/src/main/java/org/springframework/security/config/websocket/WebSocketMessageBrokerSecurityBeanDefinitionParser.java. 
But we can reconsider.

Although we have reflected that in our [Code Style](https://github.com/spring-projects/spring-integration/wiki/Spring-Integration-Framework-Code-Style) rules already :smile: : 

CC @garyrussell